packages/hash/integration/src/mockData/index.ts

```diff
@@ -437,22 +438,11 @@ void (async () => {
                 "employer.entityType",
               ],
             },
-            data: {
-              __linkedData: {
-                entityTypeId: newTypeIds.Person,
-                aggregate: {
-                  itemsPerPage: 5,
-                  multiSort: [
-                    {
-                      field: "createdAt",
-                    },
-                  ],
-                },
-              },
-            },
+            data: {},
           },
         },
       ],
+      // @todo: should this block be removed?
```

Comment:
The table block defined there is kind of dysfunctional because of the linkedAggregation changes. That is, it isn't displaying anything yet.

Hence, we can either remove it or create a task for fixing it later on.

The table was displaying collab editing functionality before.

Reply:
Can you expand the comment to explain this please?

Reply:
Done.